### ISSUE TITLE
imgui-sfml IM_ASSERT failure

### ISSUE THREAD

**Issue by esduran** [microsoft/vcpkg#9660]
**Describe the bug**
When running my application in x64-Debug I get:
`Assertion failed: g.FrameScopeActive, file C:\vcpkg\buildtrees\imgui\src\v1.73-a85d24b4b8\imgui.cpp, line 5400`
only when I add `ImGui::Begin` and `ImGui::End` calls to my source file.

I am attempting to start a new C++ project using vcpkg, CMake, and Visual Studio on windows. I installed the following packages through vcpkg:

-  imgui-sfml:x64-windows    v2.1
- imgui:x64-windows   v1.73-1
- sfml:x64-windows   v2.5.1-4

In my _CMakeLists.txt_ I include the following:
```
...
find_package(SFML CONFIG COMPONENTS REQUIRED windows graphics system)
find_package(imgui CONFIG REQUIRED)
find_package(ImGui-SFML CONFIG REQUIRED)
...
add_executable(main app/main.cpp)
target_link_libraries(main
                                PRIVATE imgui::imgui
                                              ImGui-SFML::ImGui-SFML
                                             sfml-system
                                             sfml-window
                                             sfml-graphics)
```

My _main.cpp_ file consists of the following:
```c++
#include <imgui.h>
#include <imgui-SFML.h>

#include <SFML/Graphics/RenderWindow.hpp>
#include <SFML/System/Clock.hpp>
#include <SFML/Window/Event.hpp>
#include <SFML/Graphics/CircleShape.hpp>

int main()
{
    sf::RenderWindow window(sf::VideoMode(640, 480), "ImGui + SFML = <3");
    window.setFramerateLimit(60);
    ImGui::SFML::Init(window);

    sf::CircleShape shape(100.f);
    shape.setFillColor(sf::Color::Green);

    sf::Clock deltaClock;
    while (window.isOpen()) {
        sf::Event event;
        while (window.pollEvent(event)) {
            ImGui::SFML::ProcessEvent(event);

            if (event.type == sf::Event::Closed) {
                window.close();
            }
        }

        ImGui::SFML::Update(window, deltaClock.restart());

        ImGui::Begin("Hello, world!");
        ImGui::Button("Look at this pretty button");
        ImGui::End();

        window.clear();
        window.draw(shape);
        ImGui::SFML::Render(window);
        window.display();
    }

    ImGui::SFML::Shutdown();
}
}
```

When running my application in Visual Studio using the x64-Debug configuration I immediately get the following assertion failure for `IM_ASSERT(g.FrameScopeActive)`. This only happens when I add the lines
```
ImGui::Begin("Test Begin");
ImGui::Button("Test Button");
ImGui::End();
```

Without them I get a running black window (first image below).

I have been unable to find a solution to this problem. I am unsure if this is an issue with ImGui-SFML or ImGui itself. Any help with this would be greatly appreciated. Thank you!

**Screenshots/Video**
![image](https://user-images.githubusercontent.com/12473592/71820810-f2ad0480-3055-11ea-97df-1674c7a679f9.png)
![error_2](https://user-images.githubusercontent.com/12473592/71820670-71557200-3055-11ea-8eca-6f7b85347805.png)

**Environment**
- OS: Windows 10
- Compiler: Visual Studio Community 2019 with MSVC v142 - VS 2019 C++ x64/x96 build tools (v14.24)

**To Reproduce**
1. ./vcpkg install imgui-sfml:x64-windows
2. Create a basic CMakeLists with the ImGui-SFML repo's example main.cpp
3. Run the application in x64 Debug and get the IM_ASSERT failure

**Expected behavior**
Expected to see an ImGui window with a button.

**Additional context**
This issue was initially [opened ](https://github.com/eliasdaler/imgui-sfml/issues/113) directly in the ImGui-SFML repository, but after some experimentation it was determined that the issue exists in the vcpkg version.

**Comment by JackBoosY** [comment]
Hi @esduran, thanks for reporting this issue!
I cannot repro it on my machine, can you make sure your vcpkg is latest?
![image](https://user-images.githubusercontent.com/47264268/72332979-f8f64d00-36f5-11ea-980f-4a6320413d68.png)

Thanks.

**Comment by esduran** [comment]
Hi @JackBoosY, I did a complete purge of vcpkg and reinstalled the packages I listed above. I still get the same exception thrown in Visual Studio of `Exception thrown: read access violation.
g was nullptr.` for the file _imgui.cpp_ in the function `ImGui::Begin` line 5400.

**Comment by BryanTriana** [comment]
I am also getting the same error when using ImGui for SFML using vcpkg.

![Screenshot (11)](https://user-images.githubusercontent.com/26044298/74078659-1d92cb80-49fb-11ea-9be0-672083ecbf04.png)
![Screenshot (10)](https://user-images.githubusercontent.com/26044298/74078660-1ec3f880-49fb-11ea-9f8b-2f04c4e4a6e1.png)

Any idea if it gets fixed if both of them are installed manually?

**Comment by esduran** [comment]
@BryanTriana I ended up statically building SFML from source and then building ImGUI-SFML from source as well. That has been working with zero issues thus far.

**Comment by luketrevorrow** [comment]
Also getting the same issue - I will try the static building of SFML like @esduran has done.

**Comment by JackBoosY** [comment]
I'm digging deeper into this issue, which may be caused by the fact that the symbols are not reused.

**Comment by WeskerPower** [comment]
`Assertion failed: g.FrameScopeActive && "Forgot to call ImGui::NewFrame()?", file C:\vcpkg\buildtrees\imgui\src\v1.73-a85d24b4b8\imgui.cpp, line 4161
Assertion failed: ImGui::GetCurrentContext() != 0 && "Missing dear imgui context. Refer to examples app!", file C:\vcpkg\buildtrees\imgui\src\v1.73-a85d24b4b8\imgui_demo.cpp, line 197
`

imgui-sfml 
code : https://github.com/eliasdaler/imgui-sfml/blob/master/examples/main.cpp

**Comment by NikBomb** [comment]
I am having the same issue, I will try to resort to manual build...

**Comment by VincentZalzal** [comment]
I also have the same problem. Here's my hypothesis.

`ImGui::SFML::Init(window)` calls `ImGui::CreateContext()`, which initializes a global pointer `GImGui`. Since this is in the namespace `ImGui::SFML` and vcpkg builds it as a dynamic library, it means `ImGui::SFML` has its own global `GImGui`. Now, when calling `ImGui::Begin()`, `GImGui` is accessed, but since it is in the `ImGui`, it is a different global pointer, which hasn't been initialized. Ah, evil global variables... That is why static linking works: there is then a single global pointer.

I hacked something in the debugger that seemed to work. In the main, I added a call to `ImGui::SetCurrentContext(ctx)`, where ctx is actually the value of `GImGui` in the `ImGui::SFML` scope, and it works (until I find there is another global variable in ImGui...).

Problem is: I don't find a way to get the `ImGui::SFML` pointer cleanly. I think a method should be added to `ImGui::SFML` to get the current context pointer.

**Comment by NikBomb** [comment]
Hi Vincent,

For the time being the workaround for me is the manual integration described here:
https://eliasdaler.github.io/using-imgui-with-sfml-pt1/

**Comment by JackBoosY** [comment]
We are digging the cause of this issue and try to support `imgui-sfml` to generate dynamic libraries.

**Comment by dan-shaw** [comment]
It seems that there are many issues with the `imgui` library, and we want to fix this. Does anyone have an example that reproduces this issue? We aren't able to reproduce this issue on our machines.

**Comment by VincentZalzal** [comment]
I did almost the same thing as the OP. To reproduce, with VS2019 and vcpkg integration enabled:
- vcpkg install imgui-sfml  (x64)
- create empty project
- compile and run the following code (imgui-sfml hello world) in x64 Debug.
```
#include "imgui.h"
#include "imgui-SFML.h"

#include <SFML/Graphics/RenderWindow.hpp>
#include <SFML/System/Clock.hpp>
#include <SFML/Window/Event.hpp>
#include <SFML/Graphics/CircleShape.hpp>

int main()
{
    sf::RenderWindow window(sf::VideoMode(640, 480), "ImGui + SFML = <3");
    window.setFramerateLimit(60);
    ImGui::SFML::Init(window);

    sf::CircleShape shape(100.f);
    shape.setFillColor(sf::Color::Green);

    sf::Clock deltaClock;
    while (window.isOpen()) {
        sf::Event event;
        while (window.pollEvent(event)) {
            ImGui::SFML::ProcessEvent(event);

            if (event.type == sf::Event::Closed) {
                window.close();
            }
        }

        ImGui::SFML::Update(window, deltaClock.restart());

        ImGui::Begin("Hello, world!");
        ImGui::Button("Look at this pretty button");
        ImGui::End();

        window.clear();
        window.draw(shape);
        ImGui::SFML::Render(window);
        window.display();
    }

    ImGui::SFML::Shutdown();
}
```

This will assert on the call to ImGui::Begin.

**Comment by WeskerPower** [comment]
![image](https://user-images.githubusercontent.com/15333431/77990103-b1876200-72dd-11ea-8615-d9190cf8d13b.png)

`Assertion failed: ImGui::GetCurrentContext() != 0 && "Missing dear imgui context. Refer to examples app!", file C:\vcpkg\buildtrees\imgui\src\v1.73-a85d24b4b8\imgui_demo.cpp, line 197`

197
`IM_ASSERT(ImGui::GetCurrentContext() != NULL && "Missing dear imgui context. Refer to examples app!"); // Exceptionally add an extra assert here for people confused with initial dear imgui setup`


code

`#include "imgui.h"
#include "imgui-SFML.h"

#include <SFML/Graphics/RenderWindow.hpp>
#include <SFML/System/Clock.hpp>
#include <SFML/Window/Event.hpp>
#include <SFML/Graphics/CircleShape.hpp>

int main()
{
    sf::RenderWindow window(sf::VideoMode(640, 480), "ImGui + SFML = <3");
    window.setFramerateLimit(60);
    ImGui::SFML::Init(window);

    sf::CircleShape shape(100.f);
    shape.setFillColor(sf::Color::Green);

    sf::Clock deltaClock;
    while (window.isOpen()) {
        sf::Event event;
        while (window.pollEvent(event)) {
            ImGui::SFML::ProcessEvent(event);

            if (event.type == sf::Event::Closed) {
                window.close();
            }
        }

        ImGui::SFML::Update(window, deltaClock.restart());

        ImGui::ShowTestWindow();

        ImGui::Begin("Hello, world!");
        ImGui::Button("Look at this pretty button");
        ImGui::End();

        window.clear();
        window.draw(shape);
        ImGui::SFML::Render(window);
        window.display();
    }

    ImGui::SFML::Shutdown();

    return 0;
}`




Attempt 2 :
![image](https://user-images.githubusercontent.com/15333431/77990709-43dc3580-72df-11ea-835e-7afeb3067dd5.png)


![image](https://user-images.githubusercontent.com/15333431/77990774-7423d400-72df-11ea-96b2-5f9d2265e6e0.png)


![image](https://user-images.githubusercontent.com/15333431/77990830-9289cf80-72df-11ea-87e2-6738a7dd7fad.png)

![image](https://user-images.githubusercontent.com/15333431/77990914-c9f87c00-72df-11ea-9504-6104e4908b6f.png)


code
`#include "imgui.h"
#include "imgui-SFML.h"

#include <SFML/Graphics/RenderWindow.hpp>
#include <SFML/System/Clock.hpp>
#include <SFML/Window/Event.hpp>
#include <SFML/Graphics/CircleShape.hpp>

int main()
{
    sf::RenderWindow window(sf::VideoMode(640, 480), "ImGui + SFML = <3");
    window.setFramerateLimit(60);
    ImGui::SFML::Init(window);

    sf::CircleShape shape(100.f);
    shape.setFillColor(sf::Color::Green);

    sf::Clock deltaClock;
    while (window.isOpen()) {
        sf::Event event;
        while (window.pollEvent(event)) {
            ImGui::SFML::ProcessEvent(event);

            if (event.type == sf::Event::Closed) {
                window.close();
            }
        }

        ImGui::SFML::Update(window, deltaClock.restart());

        ImGui::Begin("Hello, world!");
        ImGui::Button("Look at this pretty button");
        ImGui::End();

        window.clear();
        window.draw(shape);
        ImGui::SFML::Render(window);
        window.display();
    }

    ImGui::SFML::Shutdown();
}`

**Comment by JackBoosY** [comment]
We will set imgui-sfml to only static library.

**Comment by JackBoosY** [comment]
Hi guys, I'm fixing this issue in #10840, can anyone help me to test it?

Thanks.

**Comment by luketrevorrow** [comment]
I’m happy to help. What do you need me to do? Not just a git pull against my vcpkg install... I will have to make sure I have got rid of all static builds of SFML on my machine first.

**Comment by JackBoosY** [comment]
@luketrevorrow Use my changes and rebuild imgui-sfml would be okay.

**Comment by NikBomb** [comment]
Hi all,

Do you still need testing?

Happy to help,
Nico

**Comment by WeskerPower** [comment]
Is not working for me but i also made a big mistake, turns out sfml-imgui has no support for the current sfml and it seems is not going to work for any version higher than 2.1 sfml

if im wrong about it please tell me, is not working for me and thats the only reason i can think of

**Comment by JackBoosY** [comment]
@NikBomb Yes please.

@WeskerPower Could you provide me more details?

Thanks.

**Comment by WeskerPower** [comment]
Well im using it without Cmake, according to umgui-sfml all i need to do is copy paste this code, after i install imgui-sfml with vcpkg

```
#include "imgui.h"
#include "imgui-SFML.h"

#include <SFML/Graphics/RenderWindow.hpp>
#include <SFML/System/Clock.hpp>
#include <SFML/Window/Event.hpp>
#include <SFML/Graphics/CircleShape.hpp>

int main()
{
    sf::RenderWindow window(sf::VideoMode(640, 480), "ImGui + SFML = <3");
    window.setFramerateLimit(60);
    ImGui::SFML::Init(window);

    sf::CircleShape shape(100.f);
    shape.setFillColor(sf::Color::Green);

    sf::Clock deltaClock;
    while (window.isOpen()) {
        sf::Event event;
        while (window.pollEvent(event)) {
            ImGui::SFML::ProcessEvent(event);

            if (event.type == sf::Event::Closed) {
                window.close();
            }
        }

        ImGui::SFML::Update(window, deltaClock.restart());

        ImGui::Begin("Hello, world!");
        ImGui::Button("Look at this pretty button");
        ImGui::End();

        window.clear();
        window.draw(shape);
        ImGui::SFML::Render(window);
        window.display();
    }

    ImGui::SFML::Shutdown();
}

```

then i have different issues one i notice a few days ago was that is not loading some files

```
'Imgui.exe' (Win32): Loaded 'E:\Gits\Imgui\Imgui\Debug\Imgui.exe'. Symbols loaded.
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ntdll.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\kernel32.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\KernelBase.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\imm32.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\user32.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\win32u.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\gdi32.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\gdi32full.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\msvcp_win.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ucrtbase.dll'. 
'Imgui.exe' (Win32): Loaded 'E:\Gits\Imgui\Imgui\Debug\sfml-system-d-2.dll'. Symbols loaded.
'Imgui.exe' (Win32): Loaded 'E:\Gits\Imgui\Imgui\Debug\ImGui-SFML.dll'. Symbols loaded.
'Imgui.exe' (Win32): Loaded 'E:\Gits\Imgui\Imgui\Debug\sfml-graphics-d-2.dll'. Symbols loaded.
'Imgui.exe' (Win32): Loaded 'E:\Gits\Imgui\Imgui\Debug\sfml-window-d-2.dll'. Symbols loaded.
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\advapi32.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\msvcrt.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\sechost.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\msvcp140d.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\rpcrt4.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\sspicli.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\cryptbase.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\bcryptprimitives.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ucrtbased.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\vcruntime140d.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\opengl32.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\opengl32.dll'. 
'Imgui.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\opengl32.dll'
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\combase.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\winmm.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\glu32.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\winmmbase.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\winmmbase.dll'. 
'Imgui.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\winmmbase.dll'
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\cfgmgr32.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\DXCore.dll'. 
'Imgui.exe' (Win32): Loaded 'E:\Gits\Imgui\Imgui\Debug\freetyped.dll'. Symbols loaded.
'Imgui.exe' (Win32): Loaded 'E:\Gits\Imgui\Imgui\Debug\zlibd1.dll'. Symbols loaded.
'Imgui.exe' (Win32): Loaded 'E:\Gits\Imgui\Imgui\Debug\bz2d.dll'. Symbols loaded.
'Imgui.exe' (Win32): Loaded 'E:\Gits\Imgui\Imgui\Debug\libpng16d.dll'. Symbols loaded.
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\uxtheme.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\msctf.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\oleaut32.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\System32\DriverStore\FileRepository\nv_dispi.inf_amd64_830a0263f2ee97ce\nvoglv32.dll'. 
'Imgui.exe' (Win32): Unloaded 'C:\Windows\System32\DriverStore\FileRepository\nv_dispi.inf_amd64_830a0263f2ee97ce\nvoglv32.dll'
'Imgui.exe' (Win32): Loaded 'C:\Windows\System32\DriverStore\FileRepository\nv_dispi.inf_amd64_830a0263f2ee97ce\nvoglv32.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\shell32.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\SHCore.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\windows.storage.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\profapi.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\powrprof.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\umpdc.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\shlwapi.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\kernel.appcore.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\cryptsp.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\setupapi.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\bcrypt.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ole32.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\wtsapi32.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\version.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\devobj.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\wintrust.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\msasn1.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\crypt32.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ntmarta.dll'. 
The thread 0x3aa4 has exited with code 0 (0x0).
The thread 0x3884 has exited with code 0 (0x0).
The thread 0x36d4 has exited with code 0 (0x0).
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\dwmapi.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\nvspcap.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\winsta.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\dinput8.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\InputHost.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\CoreMessaging.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\CoreUIComponents.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\WinTypes.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\propsys.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\hid.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\TextInputFramework.dll'. 
'Imgui.exe' (Win32): Loaded 'C:\Windows\SysWOW64\iertutil.dll'. 
Exception thrown: read access violation.
**g** was nullptr.

The program '[6028] Imgui.exe' has exited with code 0 (0x0).
```

```
Severity	Code	Description	Project	File	Line	Suppression State	Detail Description
Warning	C6011	Dereferencing NULL pointer 'Buf.Data'. 	Imgui	C:\vcpkg\installed\x86-windows\include\imgui.h	1639	

```

```
Imgui.exe!ImGui::Begin(const char * name, bool * p_open, int flags) Line 5248	C++

Imgui.exe!main() Line 31	C++



  | Name | Value | Type
-- | -- | -- | --
▶ | *GImGui | <struct at NULL> | ImGuiContext


  | Name | Value | Type
-- | -- | -- | --
▶ | g | <struct at NULL> | ImGuiContext &

```



Im not sure if im doing something wrong over here

**Comment by JackBoosY** [comment]
@WeskerPower Did you use #10840 changes?

**Comment by WeskerPower** [comment]
> @WeskerPower Did you use #10840 changes?

i just replace or created those files and didnt work for me, should i do anything else ?

**Comment by JackBoosY** [comment]
@WeskerPower You should clone my repo `https://github.com/JackBoosY/vcpkg.git` and switch to branch `dev/jack/9660`.

**Comment by WeskerPower** [comment]
`

> @WeskerPower You should clone my repo `https://github.com/JackBoosY/vcpkg.git` and switch to branch `dev/jack/9660`.

Thanks ok i had a bit of trouble there, but i manage to do what i needed to do, and now its working

![image](https://user-images.githubusercontent.com/15333431/79957864-d9369980-8447-11ea-9cc0-70623c38d783.png)


i can confirm that solves the issue of g been null

**Comment by NikBomb** [comment]
Dear All, I can confirm that the changes fixe the issue.

**Comment by JackBoosY** [comment]
Fixed by #11800.
